### PR TITLE
[doc]update yield -EN

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Update yield syntax -EN

### CODE REVIEW

File: docs/manual-EN/statement-syntax/data-manipulation-statements/yield-syntax.md [hunk 1]
```diff
@@ -1,46 +1,100 @@
-# Yield Syntax
+# Yield Clause and Statement
 
+ Keyword `YIELD` can be used as a clause in a `FETCH` or `GO` statement, or as a separate statement in `PIPE`, or as a single sentence for calculation.
+
+## As Clause
```
Comment (whitewum) [inline docs/manual-EN/statement-syntax/data-manipulation-statements/yield-syntax.md]: ## As Clause (with Go-syntax)

File: docs/manual-EN/statement-syntax/data-manipulation-statements/yield-syntax.md [hunk 2]
```diff
@@ -1,46 +1,100 @@
...
+(user@127.0.0.1) [myspace_test2]> YIELD hash("Tim") % 100
+=====================
+| (hash("Tim")%100) |
+=====================
+| 42                |
+---------------------
 ```
 
-e.g., $$.student.name is to get the target ($$) vertex (student)'s property (name). 
+## Note
```
Comment (whitewum) [inline docs/manual-EN/statement-syntax/data-manipulation-statements/yield-syntax.md]: ### Note

File: docs/manual-EN/statement-syntax/data-manipulation-statements/yield-syntax.md [hunk 3]
```diff
@@ -1,46 +1,100 @@
...
 
-* DISTINCT
+- DISTINCT
```
Comment (whitewum) [inline docs/manual-EN/statement-syntax/data-manipulation-statements/yield-syntax.md]: delete this line

File: docs/manual-EN/statement-syntax/data-manipulation-statements/yield-syntax.md [hunk 4]
```diff
@@ -1,46 +1,100 @@
...
 
-`YIELD DISTINCT` must be used with the `GO` statement.
-```
+YIELD DISTINCT is not supported for use in single sentences.
```
Comment (whitewum) [inline docs/manual-EN/statement-syntax/data-manipulation-statements/yield-syntax.md]: you can not use `YIELD DISTINCT` as a separate statement. This is a  syntax error.

**Comment by Amber1990Zhang** [comment]
> If you're translating from or to the other language, please ref the other pr (`#123`), or ref the other md in the decription.
> So can i check your meanings are correct.

Sorry for the inconvenience, translated from pr #1065

**Pull request merged**